src/gui/windows/multiplayer.zig

```diff
@@ -56,14 +56,17 @@ fn join(_: usize) void {
 		ipAddress = "";
 	}
 	if(connection) |_connection| {
+		if (ipAddressEntry.currentString.items.len == 0) {
+			// TODO: Show error message in the GUI
```

Comment:
see #810

Reply:
added in 55686fb0bf1275e4edb400f83523fb594255f4a2